### ISSUE TITLE
Detective doesn't detect keys as extra keys if a read input has been added for keys added in template comments

### ISSUE THREAD

**Issue by dorthrithil** [ngneat/transloco-keys-manager#58]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I started with the following template:

```
<!-- t(de, en, zh) -->
```

After extracting keys, the following keys were added correctly:

```
  "de": "Missing value for 'de'",
  "en": "Missing value for 'en'",
  "zh": "Missing value for 'zh'",
```

Now I changed the template to:

```
<ng-container *transloco="let t; read: 'accountSettings'">
<!-- t(de, en, zh) -->
</ng-container>
```

Running key extraction resulted in:

```
  "accountSettings.de": "Missing value for 'accountSettings.de'",
  "accountSettings.en": "Missing value for 'accountSettings.en'",
  "accountSettings.zh": "Missing value for 'accountSettings.zh'",
  "de": "Missing value for 'de'",
  "en": "Missing value for 'en'",
  "zh": "Missing value for 'zh'",
```

Now comes the error: Running `find` doesn't yield any extra keys. However `de`, `en` and `zh` should have been detected. 

A similar scenario where no comment keys are used but the `{{ t('de') }}` syntax, the detective correctly recognises the old keys as extra keys.

## Expected behavior

`de`, `en` and `zh` should have been detected as extra keys. 

## Minimal reproduction of the problem with instructions

See steps above.

## Environment

- Node version: 13.7.0
- Platform:  Mac

**Comment by dorthrithil** [comment]
Just also noticed, that after manually deleting the extra keys and running `extract` again the keys are added back in. So the tool is actually creating them twice. Once prefixed, once without prefix.

**Comment by Alegiter** [comment]
+1
Duplicate of [50#](https://github.com/ngneat/transloco-keys-manager/issues/50#)

**Comment by dorthrithil** [comment]
It's not exactly a duplicate as #50 doesn't mention the detective bug - but I guess it's closely related.

**Comment by Alegiter** [comment]
Sorry, agreed with you) 
I think if #50 will be solved, then this issue will be solved too. maybe)

**Comment by shaharkazaz** [comment]
@dorthrithil Was this issue resolved via the fix in #50?

**Comment by dorthrithil** [comment]
Yes, the error is gone. Thank you!
